src/views/DidUpgradeExplainer/DidUpgradeExplainer.tsx

```diff
@@ -33,14 +59,27 @@ export function DidUpgradeExplainer({ identity }: Props): JSX.Element {
       </div>
       <p className={styles.deposit}>{t('view_DidUpgradeExplainer_deposit')}</p>
 
-      <Link
-        to={generatePath(paths.identity.did.upgrade.sign, {
-          address: identity.address,
-        })}
-        className={styles.upgrade}
-      >
-        {t('view_DidUpgradeExplainer_CTA')}
-      </Link>
+      {is_active && remaining_dids > 0 && (
```

Comment:
Do you think we should check both of them?

Reply:
Should I check with Albi if `is_active` will always be false when `remaining_dids` is 0?